mysql/errcode.go

```diff
@@ -939,6 +939,7 @@ const (
 	ErrSpatialFunctionalIndex                                       = 3760
 	ErrWrongKeyColumnFunctionalIndex                                = 3761
 	ErrFunctionalIndexOnField                                       = 3762
+	ErrReferencedForeignKey                                         = 3780
```

Comment:
According to the MySQL [document](https://dev.mysql.com/doc/refman/8.0/en/server-error-reference.html),
> Error number: 3780; Symbol: ER_FK_INCOMPATIBLE_COLUMNS; SQLSTATE: HY000
Message: Referencing column '%s' and referenced column '%s' in foreign key constraint '%s' are incompatible.

The error name should be `ErrFKIncompatibleColumns`.

Reply:
great, done.